src/widgets/radio.jsx

```diff
@@ -417,41 +516,35 @@ var RadioEditor = React.createClass({
         return <div>
             <div className="perseus-widget-row">
 
-                <div>
-                    <div className="perseus-widget-left-col">
+                <div className="perseus-widget-left-col">
+                    <div>
                         <PropCheckBox label="One answer per line"
                                       labelAlignment="right"
                                       onePerLine={this.props.onePerLine}
                                       onChange={this.props.onChange} />
+                        <InfoTip>
+                            <p>
+                                Use one answer per line unless your question has
+                                images that might cause the answers to go off the
+                                page.
+                            </p>
+                        </InfoTip>
                     </div>
-                    <InfoTip>
-                        <p>
-                            Use one answer per line unless your question has
-                            images that might cause the answers to go off the
-                            page.
-                        </p>
-                    </InfoTip>
                 </div>
 
-                <div className="perseus-widget-left-col">
+                <div className="perseus-widget-right-col">
                     <PropCheckBox label="Multiple selections"
                                   labelAlignment="right"
                                   multipleSelect={this.props.multipleSelect}
                                   onChange={this.onMultipleSelectChange} />
                 </div>
 
-                <div className="perseus-widget-right-col">
+                <div className="perseus-widget-left-col">
```

Comment:
Could you reorder these two so they appear in the same order in the source and on the page?